packages/ibm-products/src/components/AboutModal/AboutModal.js

```diff
@@ -221,7 +221,7 @@ AboutModal.propTypes = {
    * IBM 8-bar logo represented alongside the IBM Services logotype.
    * Please follow these guidelines to ensure proper execution.
    */
-  title: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired,
```

Comment:
Do we want to only strictly accept strings? `PropTypes.node` will allow consumers to pass either a string or a node and gives more flexibility. I believe this was discussed a while back during the initial release review for `AboutModal`, [here](https://github.com/carbon-design-system/ibm-products/issues/385).

[Carbon's ModalHeader component](https://react.carbondesignsystem.com/?path=/docs/components-composedmodal--overview) `title` prop is also specified as `PropTypes.node`.

Reply:
As we talked over Slack Huddle, going to move this back to using `node`, and use a dropdown to show _short_ and _long_ titles